app/forms/funding/bursary_form.rb

```diff
@@ -4,14 +4,26 @@ module Funding
   class BursaryForm < TraineeForm
     FIELDS = %i[
       applying_for_bursary
+      bursary_tier
     ].freeze
 
     attr_accessor(*FIELDS)
 
+    before_validation :set_applying_for_bursary
+
     validates :applying_for_bursary, inclusion: { in: [true, false] }
+    validates :bursary_tier, inclusion: { in: Trainee.bursary_tiers.keys }, allow_blank: true
```

Comment:
If the user is on the tiered bursary form, we shouldn't allow them to submit without something selected. If the form wasn't pre-filled with the "no bursary" option (see above), then we would be incorrectly allowing this.

Reply:
Fixed in 19942c6c04e19833ef60f8febb72b09e8b15009a

Reply:
This isn't quite fixed yet - I can still submit an empty bursary form. We need to show the form in error if nothing is selected.

This is what it looks like for the other bursary form, I think we can use the same error content (cc @edwardhorsford in Greg's absence):

<img width="674" alt="Screenshot 2021-07-12 at 11 19 00" src="https://user-images.githubusercontent.com/18436946/125271730-fa143480-e302-11eb-92ef-ff4f59219737.png">

Reply:
That makes sense, ensured that the form won't default to false, and added the same error message as the standard bursary form.

Reply:
yeah, should work similarly as for the other bursary form. The error message looks generic enough that it can be reused.